// fonts/defaultFont.js
module.exports = {
	" ": ["   ", "   ", "   "],	
	".": ["  ", "  ", ". "],
	"_": ["   ", "   ", "___"],
	"+": ["   ", "_|_", " | "],
	"-": ["   ", "___", "   "],
	"!": ["  / ", " /  ", ".   "],
	"?": ["__  ", " _] ", " .  "],
	":": ["   ", ".  ", ".  "],
	";": ["   ", ".  ", ",  "],
	",": ["   ", "   ", ",  "],
	"0": [" _  ", "| | ", "|_| "],
	"1": ["    ", "/|  ", "_|_ "],
	"2": [" __  ", " __] ", "[__  "],
	"3": ["____ ", "___| ", "___| "],
	"4": ["     ", "|__| ", "   | "],
	"5": [" __  ", "[__  ", " __] "],
	"6": ["     ", "|__  ", "|__| "],
	"7": [" __  ", "  /  ", " /   "],
	"8": [" __  ", "|__| ", "|__| "],
	"9": [" __  ", "|__| ", "   | "],
	a: ["____ ", "|__| ", "|  | "],
	b: ["___  ", "|__] ", "|__] "],
	c: [ "____ ", "|    ", "|___ "],
	d: [ "___  ", "|  \\ ", "|__/ "],
	e: ["____ ", "|___ ", "|___ "],
	f: ["____ ", "|___ ", "|    "],
	g: ["____ ", "| __ ", "|__] "],
	h: ["_  _ ", "|__| ", "|  | " ],
    i: ["_ ", "| ", "| "],
    j: [" _ ",  " | ", "_| " ],
    k: ["_  _ ", "|_/  ", "| \\_ "],
    l: ["_    ", "|    ", "|___ "],
    m: ["_  _ ", "|\\/| ", "|  | "],
    n: ["_  _ ", "|\\ | ", "| \\| "],
    o: ["____ ", "|  | ", "|__| "],
    p: ["___  ", "|__] ", "|    "],
    q: ["____ ", "|  | ", "|_\\| "],
    r: ["____ ", "|__/ ", "|  \\ "],
	s: ["____ ", "[__  ", "___] "],
    t: ["___ ", " |  ", " |  "],
	u: ["_  _ ", "|  | ", "|__| "],
    v: ["_  _ ", "|  | ", " \\/  "],
    w: ["_ _ _ ", "| | | ", "|_|_| "],
    x: ["_  _ ", " \\/  ", "_/\\_ "],
    y: ["_   _ ", " \\_/  ", "  |   "],
    z: ["___  ", "  /  ", " /__ "]
};
